distributions/standalone/src/main/resources/conf/scheduler-pools.conf

```diff
@@ -0,0 +1,27 @@
+# Configures the thread pools of the Mule Runtime
+# Certain attributes support expressions. Those are arithmetic expressions (+, -, / or * operators supported only) that are
+# evaluated when the runtime starts.
+# Those expressions may be applied to numbers (integers or decimals) or any of the following variables:
+# * C: the number of processors available to the Java virtual machine.
+
+# The maximum time to wait until all tasks in all the runtime thread pools have completed execution when stopping the
+# scheduler service.
+SchedulerService.gracefulShutdownTimeoutSeconds=15
```

Comment:
Is this case sensitive? Typically property files use lower case for keys. Like, for example, what quartz does `org.quartz.threadPool.threadCount`